lib/src/facts/external/windows/powershell_resolver.cc

```diff
@@ -1,18 +1,24 @@
 #include <internal/facts/external/windows/powershell_resolver.hpp>
+#include <internal/util/yaml.hpp>
 #include <facter/facts/collection.hpp>
 #include <facter/facts/scalar_value.hpp>
 #include <facter/facts/fact.hpp>
 #include <leatherman/windows/system_error.hpp>
 #include <leatherman/windows/windows.hpp>
 #include <leatherman/execution/execution.hpp>
 #include <leatherman/logging/logging.hpp>
+#include <leatherman/util/strings.hpp>
 #include <boost/algorithm/string.hpp>
 #include <boost/filesystem.hpp>
 
 using namespace std;
+using namespace boost::filesystem;
 using namespace leatherman::execution;
+using namespace leatherman::util;
 using namespace leatherman::windows;
-using namespace boost::filesystem;
+using namespace facter::util::yaml;
+
+namespace lth = leatherman::util;
```

Comment:
This is redundant with `using namespace leatherman::util`. A pattern used in pxp-agent is to avoid `using` statements, and do `namespace lth_util = leatherman::util;`. That makes understanding where functions come from easier. Should at least pick one approach.